### ISSUE TITLE
Many scripts not available on secure screen

### ISSUE THREAD

**Issue by CyrilleB79** [nvaccess/nvda#14286]
### Steps to reproduce:
You should have already configured NVDA to be used on Windows secure screen.
1. Start the computer to reach the logon screen
2. Press NVDA+N
3. Press NVDA+1
4. etc.
### Actual behavior:
2. No NVDA menu
3. Help mode is not enabled
### Expected behavior:
2. NVDA menu should open.
3. Help mode should be enabled
### NVDA logs, crash dumps and other attachments:
N/A
#### NVDA installed/portable/running from source:
installed
#### NVDA version:
2022.4beta2
#### Windows version:
Windows 10 2004 (x64) build 19041.388
But same issue seen on other versions of Windows 10.
#### Name and version of other software in use when reproducing the issue:
N/A
#### Other information about your system:
N/A
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Not tried.
But I remember that the issue was not occurring before. I think it has arrived with NVDA2022.4beta2.
#### If NVDA add-ons are disabled, is your problem still occurring?
Not tested.
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by seanbudd** [comment]
Thanks for creating this issue @CyrilleB79. We are aware of this problem introduced in 2022.3.1 and have begun work on fixing it.

**Comment by CyrilleB79** [comment]
On a French mailing list, there is also a [message](https://groups.io/g/nvda-fr/topic/nvda_2022_3_1_et_clavier/94530692?p=,,,100,0,0,0::recentpostdate/sticky,,,100,2,0,94530692,previd%3D1666676286283460360,nextid%3D1660631749187859350&previd=1666676286283460360&nextid=1660631749187859350
) from someone complaining not to be able to use braille keyboard on the logon screen to enter his password. The issue has appeared with NVDA 2022.3.1.
I have no more information.

Could this be due to the same cause?
